doc/virtualization.rst

```diff
@@ -319,6 +313,41 @@ that such resource allocation will make GPU basically unusable from the host.
             Kernel driver in use: vfio-pci
             Kernel modules: i915, intel_vsec
 
+Consider that not all resources can be given to VMs. Some resources are used by PF (for example, for FW).
+Also, kernel mode driver keeps some resources reserved for PF.
+
+You can further configure resource allocation for VMs in 2 ways:
+
+* Increase resources reserved for PF which will eventually reduce what you will be able to use for VMs::
+
+    sudo su
+    CARD=/sys/class/drm/card1
+
+    echo BYTES > $CARD/prelim_iov/pf/gt/lmem_spare
+    # consider other "*_spare" resources...
+
+* Explicitly allocate resources for each VM::
+
+    sudo su
+    CARD=/sys/class/drm/card1
+
+    echo BYTES > $CARD/prelim_iov/vf1/gt/lmem_quota
+
+Kernel mode driver will reject too high or too low resource values.
```

Comment:
too high or too low resource values ->
resource values that are too high or too low

Reply:
done